specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerAppsJobs_ListByResourceGroup.json

```diff
@@ -0,0 +1,100 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "api-version": "2022-11-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/containerAppsJobs/testcontainerAppsJob0",
+            "name": "testcontainerAppsJob0",
+            "type": "Microsoft.App/containerAppsJobs",
+            "location": "East US",
+            "properties": {
+              "provisioningState": "Succeeded",
```

Comment:
Can we remove this and add the change with the status API changes?

Reply:
I think this is required for all ARM resources: https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc005-provisioning-state-semantics-must-be-followed

Reply:
@trajkobal provisioning state is required, hence keeping this.